app/src/main/res/values/strings.xml

```diff
@@ -24,7 +24,16 @@
     <string name="give_answer_here_hint">Entrez ici votre réponse</string>
     <string name="qedit_logo_description">QEDit Logo</string>
     <string name="textview">TextView</string>
-    <string name="delete">delete</string>
+    <string name="delete">Delete</string>
     <string name="edit">Edit</string>
     <string name="title_activity_edit_quiz">Édition</string>
+
+    <string name="NO_DUPLICATE">"Can't have duplicate names"</string>
```

Comment:
Can we use kebab case for all string identifiers ? I kinda hurts my eyes to see two different styles in the same file...

Reply:
> Can we use kebab case for all string identifiers ? I kinda hurts my eyes to see two different styles in the same file...

I think you mean the snake case ;)